torchao/prototype/safetensors/safetensors_support.py

```diff
@@ -136,8 +150,11 @@ def save_tensor_state_dict(
         combined_tensors_dict.update(prefixed_tensors_dict)
 
     combined_metadata["tensor_names"] = json.dumps(list(tensor_dict.keys()))
+    if do_save_file:
```

Comment:
instead of adding the boolean flag, might be better to just change the name of the function and just use this to get `combined_tensors_dict, combined_metadata`? we can also test `save_file` outside of this util function